### ISSUE TITLE
LLVM ERROR: IO failure on output stream: Invalid argument

### ISSUE THREAD

**Issue by darkdh** [brave/brave-browser#2051]
Consistently reproducible in Release build only on MacOS since C71
```
Traceback (most recent call last):
  File "../../build/toolchain/mac/linker_driver.py", line 229, in <module>
    Main(sys.argv)
  File "../../build/toolchain/mac/linker_driver.py", line 85, in Main
    linker_driver_outputs += linker_driver_actions[name](args)
  File "../../build/toolchain/mac/linker_driver.py", line 119, in <lambda>
    lambda full_args: action(sub_arg[len(name):], full_args))
  File "../../build/toolchain/mac/linker_driver.py", line 145, in RunDsymUtil
    subprocess.check_call(['xcrun', 'dsymutil', '-o', dsym_out, linker_out])
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['xcrun', 'dsymutil', '-o', './Brave Browser Framework.dSYM', 'obj/chrome/chrome_framework_shared_library/Brave Browser Framework']' returned non-zero exit status 1
```

**Comment by garrettr** [comment]
In case this is helpful, here are the different versions of dsymutil for Xcode 10.1 (latest version, release build fails with llvm error) and Xcode 9.2 (downgraded version known to build without llvm error):

```sh
# Xcode 10.1
Wed Dec 05 12:32:21 ~/brave (master *)$ /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/dsymutil -version
Apple LLVM version 10.0.0 (clang-1000.11.45.5)
  Optimized build.
  Default target: x86_64-apple-darwin18.2.0
  Host CPU: skylake
# Xcode 9.2
Wed Dec 05 12:32:32 ~/brave (master *)$ find /Applications/Xcode9.2.app/ -name "dsymutil"
/Applications/Xcode9.2.app//Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/dsymutil
Wed Dec 05 12:33:09 ~/brave (master *)$ /Applications/Xcode9.2.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/dsymutil -version
Apple LLVM version 9.0.0 (clang-900.0.39.2)
  Optimized build.
  Default target: x86_64-apple-darwin18.2.0
  Host CPU: skylake
```

**Comment by bridiver** [comment]
works with `--official_build=false`

**Comment by hieupm37** [comment]
This will help: https://bugs.chromium.org/p/chromium/issues/detail?id=780980

**Comment by darkdh** [comment]
confirmed it is fixed with 74.0.3728.0
Thanks @hieupm37
